Sources/Build/BuildPlan.swift

```diff
@@ -578,19 +578,22 @@ public final class SwiftTargetBuildDescription {
         // Do nothing if we're not generating a bundle.
         guard let bundlePath = self.bundlePath else { return }
 
-        // Compute the basename of the bundle.
-        let bundleBasename = bundlePath.basename
-
         let stream = BufferedOutputByteStream()
         stream <<< """
         import class Foundation.Bundle
 
         extension Foundation.Bundle {
             static var module: Bundle = {
-                let bundlePath = Bundle.main.bundlePath + "/" + "\(bundleBasename)"
-                guard let bundle = Bundle(path: bundlePath) else {
-                    fatalError("could not load resource bundle: \\(bundlePath)")
+                let mainPath = Bundle.main.bundlePath + "/" + "\(bundlePath.basename)"
+                let buildPath = "\(bundlePath.pathString)"
+
+                let preferredBundle = Bundle(path: mainPath)
+                let fallBackBundle = Bundle(path: buildPath)
```

Comment:
Since this could be a little expensive, it might be worth instantiating this second Bundle only if the first one can't be found, especially since at deployment time for the non-test products, the first one should always be found.

Reply:
That's fair; I've inlined the instantiation of this in the ternery expression per this feedback